modules/test/playwright/helpers/HeadlessAdminUserApiHelper.ts

```diff
@@ -147,6 +165,16 @@ export class HeadlessAdminUserApiHelper {
 		);
 	}
 
+	async postRoleByERCUserAccountAssociation(
```

Comment:
please use the extended version because in other places we are using it (like deleteSiteDocumentsFolderByExternalReferenceCode)

so postRoleByExternalReferenceCodeUserAccountAssociation